README.md

```diff
@@ -1,62 +1,29 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Portfolio
 
-> Description the project.
+> For the first milestone in the process of creating my portfolio website:
 
+1. I set up a new repository and prepare it for development using best practices (e.g. linters).
+2. I build the first 2 sections of the mobile website using the template I chose in the previous step.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML & CSS
 
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
-
-## Getting Started
-
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
-
-## Authors
+## Author
 
 👤 **Author1**
```

Comment:
- [ ] Let's change 👤 **Author1** to  👤 **Ope Solomon** , so that your readme looks more professional.